tst/testbugfix/2017-06-20-blist-list-len.tst

```diff
@@ -0,0 +1,17 @@
+# Actual bug -- checking lists are the same length
```

Comment:
What does "Actual bug" mean here exactly? Doesn't seem to convey any useful/helpful information :-(